bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/Patientenblatt2.java

```diff
@@ -575,6 +581,34 @@ public void widgetSelected(SelectionEvent e) {
 			}
 		});
 		increasedTreatmentBtn.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 2, 1));
+		
+		activateMediOrderBtn = tk.createButton(cPersonalien, Messages.Patientenblatt2_activatePatient, SWT.CHECK);
+		activateMediOrderBtn.addSelectionListener(new SelectionAdapter() {
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+				if (actPatient != null) {
+					IPatient patient = ContextServiceHolder.get().getActivePatient().orElse(null);
+					if (activateMediOrderBtn.getSelection()) {
+						Stock stock = new Stock(PATIENT_STOCK + patient.getPatientNr(), 0); // $NON-NLS-1$
```

Comment:
das Soll in `IStockService` implementiert werden, bitte dazu auch einen Test schreiben!